lib/subspace/commands/init.rb

```diff
@@ -57,11 +57,11 @@ def run
       - ensure the correct roles are present in #{@env}.yml
       - Check ansible configuration variables in group_vars/#{@env}
 
-    4. Provision the new servers with ansible:
+    5. Provision the new servers with ansible:
 
       subspace provision #{@env}
 
-    !!MAKE SURE YOU PUT config/subspace/subspace.pem SOMEWHERE!!
+    !!MAKE SURE YOU PUT config/subspace/subspace.pem in 1Password!!
```

Comment:
let's update this to say that you should also add the pem/ssh key passphrase into 1PW (if you chose to add a passphrase instead of leaving it blank when prompted)